assets/questions/rle.json

```diff
@@ -92,7 +94,7 @@
         "acquired_skills": ["String Manipulation", "Sets", "Arrays", "Maps"],
         "input_function": null,
         "output_function": null,
-        "main_function": "sample.DecodeEncodedString",
+        "main_function": "DecodeEncodedString",
```

Comment:
That's my bad for being confusing :)

Here, the student is expected to write encode. For this test, however, we're asking them to make sure that their encoded string can be properly decoded, and then we're running _our_ Decode method on the result of their encode, and making sure the input == decode(encode(input)). I realized that wouldn't be in sample, so I dropped it. It's provided in aux_code.